js/lib/widget.js

```diff
@@ -316,6 +316,9 @@ export class RemoteFrameBufferView extends DOMWidgetView {
         if (!this.el.style.width && this.model.get('css_width')) {
             this.el.style.width = this.model.get('css_width');
             this.el.style.height = this.model.get('css_height');
+            // prevent massive size due to auto-scroll (issue #62)
+            this.el.style.maxWidth = "1536x";
+            this.el.style.maxHeight = "1024px";
```

Comment:
Allow at least 1024x1024, or larger, based on the size of the browser window at the time the canvas is created.
```suggestion
            this.el.style.maxWidth = Math.max(1024, window.innerWidth) + 'px';
            this.el.style.maxHeight = Math.max(1024, window.innerHeight) + 'px';
```